### PR TITLE
launch.json and container configuration

### LINKED ISSUE #513: Create a launch.json with relative paths
- Commands:
  - Generation for language/endpoint pairs
  - TestData generation for language/endpoint pairs
- Paths need to be relative
- Should work cross-platform (and in containers)
- Update README regarding these changes. 
[AB#8978](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/8978)

### PR DESCRIPTION
**Pull request by zengin**
Fixes #513 

- Adds a script to generate `launch.json` to handle the repetition in the JSON.
- Adds the updated `launch.json` which only uses relative paths. It should be ready to use by anyone checking out `msgraph-metadata` and `MSGraph-SDK-Code-Generator` into the same root folder.
- Adds `devcontainer.json` for developing in GitHub codespaces and VSCode remote containers.
- Tested the debug experience in Windows (local box) and in Ubuntu (GitHub codespaces).

Follow up will be #514 to cover publishing changes into an SDK repo.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/pull/515)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,6 +82,10 @@ The output C# code files will go in to the `output` directory.
 
 `.\typewriter.exe -v Info -m D:\v1.0_2018_10_23_source.xml -o output -l CSharp -d D:\repos\microsoft-graph-docs -g Full`
 
+## Use/debug Typewriter in VSCode
+- Prerequisite is that `msgraph-metadata` is checked out at the same root folder as `MSGraph-SDK-Code-Generator`.
+- `launch.json` file contains main generation scenarios for each language/endpoint pair. Running Typewriter using one of those generation scenarios (e.g. `Generate CSharp beta`) will output the corresponding models into a folder, which will be a sibling to `MSGraph-SDK-Code-Generator`. The folder will be named as `generator-output-<language>-<endpoint>` e.g. `generator-output-CSharp-beta`.
```
Comment (baywet) [inline README.md:87]: why don't we use the repos names/paths for the output instead of those placeholders?

**Review by baywet — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (zengin) [reply README.md:87]: See #514

Comment (baywet) [reply README.md:87]: so in that model we'd have the debug process to generate to an output folder and a post script to delete + move things, correct? I think launch.json supports adding such post scripts, would you mind adding it as well?

Comment (zengin) [reply README.md:87]: Linked this comment to be addressed as part of #514.

**Pull request merged**